src/alda/repl/commands/ReplDown.java

```diff
@@ -0,0 +1,31 @@
+package alda.repl.commands;
+
+import alda.AldaResponse.AldaScore;
+import alda.AldaServer;
+
+import java.util.function.Consumer;
+
+import jline.console.ConsoleReader;
+
+public class ReplDown implements ReplCommand {
+  public ReplDown() {}
```

Comment:
These empty constructors aren't required, but I guess it doesn't hurt to have them for readability